agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/common/NetworkFriendlyExceptions.java

```diff
@@ -22,20 +22,36 @@
 package com.microsoft.applicationinsights.agent.internal.common;
 
 import com.microsoft.applicationinsights.agent.internal.configuration.DefaultEndpoints;
+import io.netty.handler.ssl.SslHandshakeTimeoutException;
 import java.io.File;
+import java.io.IOException;
 import java.net.UnknownHostException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
+import javax.net.ssl.SSLContext;
 import javax.net.ssl.SSLHandshakeException;
+import javax.net.ssl.SSLSocketFactory;
 import org.checkerframework.checker.nullness.qual.Nullable;
 import org.slf4j.Logger;
 
 public class NetworkFriendlyExceptions {
+  public static final List<String> expectedCiphers =
```

Comment:
helps when reading the code below to know that this is immutable constant
```suggestion
  public static final List<String> EXPECTED_CIPHERS =
```